apf/apfIPShape.cc

```diff
@@ -244,6 +256,12 @@ class ConstantIPFit : public IPBase
           grads.allocate(1);
           grads[0] = Vector3(0,0,0);
         }
+        void getVectorValues(Mesh*, MeshEntity*,
+            Vector3 const&,
+            NewArray<Vector3>&) const
+        {
+          fail("getVectorValues not defined for ConstantIPFit shapes");
```

Comment:
Why not use `PCU_ALWAYS_ASSERT` here?

Reply:
This is to stay consistent with the surrounding code see for example like 163.

Reply:
ok